src/poetry/console/commands/remove.py

```diff
@@ -108,15 +109,13 @@ def handle(self) -> int:
             )
 
         # Refresh the locker
-        self.poetry.set_locker(
-            self.poetry.locker.__class__(self.poetry.locker.lock, poetry_content)
```

Comment:
There you go with the tests :) In order to make them work I had to work around this line that was effectively resetting the mocked attributes of the `TestLocker`

Reply:
If you like it I will apply it to the `AddCommand` too

Reply:
Looks better to me than before.